### PR TITLE
SPUI-724 > Display phone number after set

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart [hunk 1]
```diff
@@ -80,6 +80,8 @@ class PhoneVerificationConfirmNotifier
 
         _userInfoN.updatePhoneVerified(phoneVerified: true);
         _userInfoN.updateTwoFaStatus(enabled: true);
+        final userInfo = read(userInfoNotipod.notifier);
+        userInfo.updatePhone(state.phoneNumber);
```
Comment (eli1stark) [inline lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart:84]: userInfoN.updatePhone(state.phoneNumber);

File: lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart [hunk 2]
```diff
@@ -80,6 +80,8 @@ class PhoneVerificationConfirmNotifier
 
         _userInfoN.updatePhoneVerified(phoneVerified: true);
         _userInfoN.updateTwoFaStatus(enabled: true);
+        final userInfo = read(userInfoNotipod.notifier);
```
Comment (eli1stark) [inline lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart:83]: remove

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart:84]: done

File: lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart (hunk 2 above)
Comment (DenisMartych) [reply lib/shared/features/phone_verification/phone_verification_confirm/notifier/phone_verification_confirm_notifier.dart:83]: done

**Pull request closed without merging**